packages/runtime-plugin-hmr/src/hmr-client.ts

```diff
@@ -13,7 +13,9 @@ const hmrHost =
     : FARM_HMR_HOST || 'localhost';
 
 const socketProtocol = location.protocol === 'https:' ? 'wss' : 'ws';
-const socketHostUrl = `${hmrHost}:${hmrPort}${FARM_HMR_PATH}`;
+const socketHostUrl = FARM_HMR_PATH.startsWith('/')
```

Comment:
under what circumstances will the problem occur/lost?

Reply:
> under what circumstances will the problem occur/lost?

See #1522, socket host url is invalid in electron

Reply:
I haven't seen the specific reason why the error was reported, but I think the solution is to find out why / was killed, rather than refixing the problem in the end.

Reply:
> I haven't seen the specific reason why the error was reported, but I think the solution is to find out why `/` was killed, rather than refixing the problem in the end.
> 
> Solve problems at their root rather than patching them up

You are right, I'll try to find where inject these globals.

Reply:
If it is a problem with electron, can we solve this problem in the electron plugin?  Because I haven’t looked at the specific reason yet, I can only give you ideological suggestions.